src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -4,25 +4,31 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.Properties;
+import java.util.Scanner;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
+    private static final String USERNAME = "root";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String password;
 
     static {
         try {
             Class.forName(JDBC_DRIVER);
         } catch (ClassNotFoundException e) {
             throw new RuntimeException("Can't find SQL Driver", e);
         }
+
+        try (Scanner scanner = new Scanner(System.in)) {
+            System.out.println("Please type password:");
+            password = scanner.nextLine();
+        }
```

Comment:
let's add ability to enter login or remove this scanner feature, it feels redundant in the scope of the task